src/profile-logic/symbol-store.js

```diff
@@ -228,9 +228,13 @@ export class SymbolStore {
   _symbolProvider: SymbolProvider;
   _db: SymbolStoreDB;
 
-  constructor(dbNamePrefix: string, symbolProvider: SymbolProvider) {
+  constructor(dbNamePrefixOrDB, symbolProvider: SymbolProvider) {
     this._symbolProvider = symbolProvider;
-    this._db = new SymbolStoreDB(`${dbNamePrefix}-symbol-tables`);
+    if (typeof(dbNamePrefixOrDB) === 'string') {
+      this._db = new SymbolStoreDB(`${dbNamePrefix}-symbol-tables`);
```

Comment:
looks like you need to change the name of the variable here

Reply:
Good spot, thank you! Fixed in ccdb5e870d2c951171d7f6e0ff516520b23adf60.